### PR TITLE
Remove Now CLI and custom server deployments from /learn

### PR DESCRIPTION
**Pull request by chibicode**
While I work on rewriting [/learn](https://nextjs.org/learn), as a temporary quick-fix, I’ve modified the last Basics lesson (["Deploying a Next.js App"](https://nextjs.org/learn/basics/deploying-a-nextjs-app)) such that:

- It no longer needs to download example code because you’re just deploying ([current page](https://nextjs.org/learn/basics/deploying-a-nextjs-app/setup) - to be removed)
- It no longer talks about Now CLI ([current page](https://nextjs.org/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now) - to be removed)
- It no longer talks about hosting on a custom server ([this page](https://nextjs.org/learn/basics/deploying-a-nextjs-app/deploying-to-your-own-environment), [this page](https://nextjs.org/learn/basics/deploying-a-nextjs-app/build-and-start), [this page](https://nextjs.org/learn/basics/deploying-a-nextjs-app/run-two-instances), [this page](https://nextjs.org/learn/basics/deploying-a-nextjs-app/build-once-run-many-instances))
- Instead, it shows an instruction for ZEIT Now Git integration, which is [copied from here](https://nextjs.org/docs/deployment). I added a link to the [deployment doc](https://nextjs.org/docs/deployment) and the [ZEIT doc](https://zeit.co/docs).

### CODE REVIEW

File: pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx [hunk 1]
```diff
@@ -10,20 +10,20 @@ export const meta = {
 
 ## Deploying to ▲ZEIT Now
 
-First, [download Now](https://zeit.co/download) if you do not already have it installed.
+If you haven’t already done so, push your Next.js app to a Git provider of your choice: [GitHub](http://github.com/), [GitLab](https://gitlab.com/), or [BitBucket](https://bitbucket.org/). Your repository can be private or public.
```
Comment (timneutkens) [inline pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: I think we should probably not link to GitHub/GitLab/BitBucket here as it'd cause people to go out of the flow of setting up the deployment

Comment (chibicode) [reply pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: @timneutkens great point, removed https://github.com/zeit/next-site/pull/574/commits/0cfadacb64bfca5c52425c32bfdb6260b73dde01

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**…/deploying-to-zeit-now**](https://next-site-git-fork-chibicode-remove-cli-from-learn.zeit.now.sh/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now) | [**…deploying-a-nextjs-app**](https://next-site-git-fork-chibicode-remove-cli-from-learn.zeit.now.sh/learn/basics/deploying-a-nextjs-app) |
|:-:|:-:|
|<a href="https://next-site-git-fork-chibicode-remove-cli-from-learn.zeit.now.sh/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now">Screenshot of /learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/r1Ni7xM_xFOX6Z_T7xkluU7_4O0_.png">(view full size)</a> |<a href="https://next-site-git-fork-chibicode-remove-cli-from-learn.zeit.now.sh/learn/basics/deploying-a-nextjs-app">Screenshot of /learn/basics/deploying-a-nextjs-app</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/FMDgO56GL2E8xjZbsx7RbObUXvA_.png">(view full size)</a> |





6 deleted routes:
- **/learn/basics/deploying-a-nextjs-app/build-and-start**
- **/learn/basics/deploying-a-nextjs-app/build-once-run-many-instances**
- **/learn/basics/deploying-a-nextjs-app/deploying-to-your-own-environment**
- **/learn/basics/deploying-a-nextjs-app/finally**
- **/learn/basics/deploying-a-nextjs-app/run-two-instances**
- **/learn/basics/deploying-a-nextjs-app/setup**

Commit 1e2267d4abdd4cbddf860ab1be74c9ce76006594 (https://next-site-git-fork-chibicode-remove-cli-from-learn.zeit.now.sh).

**Comment by chibicode** [comment]
Actually, we might want to wait until https://github.com/zeit/next.js/pull/10226 is merged:

- We'd like to encourage users to deploy by pushing to GitHub.
- However, we currently tell users to `clone` `next-learn-demo` ([Example](https://nextjs.org/learn/basics/styling-components/setup)). They'd have to do a bit of work to create a separate repo for the subdirectory (e.g. `7-styling-components`) and push that to their own GitHub account.
- Instead of `git clone`, we can tell users to use `curl https://codeload.github.com/zeit/next-learn-demo/tar.gz/master | tar -xz` but then `.gitignore` won't be copied.

If we merge https://github.com/zeit/next.js/pull/10226 first, we can tell them to use `create-next-app` to download the example, and it copies `.gitignore`.

**Comment by chibicode** [comment]
Reopened since https://github.com/zeit/next.js/pull/10226 is merged

**Comment by chibicode** [comment]
https://github.com/zeit/next.js/pull/10226 is merged but this branch still needs to be updated. Closing until then.

**Pull request closed without merging**